### PR TITLE
OPENEUROPA-1766: Update ec-europa/toolkit to ~4.0.0-beta4.

### PR DESCRIPTION
**Pull request by voidtek**
### Description

Update ec-europa/toolkit to ~4.0.0-beta4.

### Change log

- Changed: Composer.json

### CODE REVIEW

**Comment by voidtek** [comment]
@jonhy81 Can you check this PR?

File: .drone.yml [hunk 1]
```diff
@@ -98,5 +98,5 @@ pipeline:
 matrix:
   PHP_VERSION:
     - 7.1
-    - 7.2
-    - 7.3
+#    - 7.2
```
Comment (jonhy81) [inline .drone.yml]: I would say to keep 7.2 as well, for sure we will upgrade PHP sooner or later.

Comment (voidtek) [reply .drone.yml]: Added.

**Comment by jonhy81** [comment]
Hi @voidtek , @ademarco any new on this PR (also in PR34)?

**Comment by ademarco** [comment]
@jonhy81 reviewing it.

**Comment by ademarco** [comment]
We got an issue when running grump on the template code itself, not one the generated subsite:

image

We are looking into this.

**Pull request merged**